### PR TITLE
New Integration - SRP Energy

### PR DESCRIPTION
**Pull request by briglx**
## Proposed change

Documentation for new integration for SRP Energy.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [x] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/41091
- Link to parent pull request in the Brands repository: https://github.com/home-assistant/brands/pull/1893
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]
We've stopped using separate pages for supporting platform types such as `sensor.srpenergy.markdown` for some time. Could you put everything together into a single `srpenergy.markdown` file? Thnx! 😄 

We also no longer use the brands map but nowadays we add logos to this repository: https://github.com/home-assistant/brands . you can delete it from the `supported_brands` folder.

I do have more comments that need to be adjusted, but let's start with the above first 😉

**Comment by briglx** [comment]
@klaasnicolaas I've removed the extra files. Looks like one was a left over from an earlier attempt.

**Comment by briglx** [comment]
@frenck can you tell if there is anything else required for this merge?

**Comment by frenck** [comment]
@briglx This awaits final review and the merge of the PR in the core codebase. The documentation will not be merged before that one.

File: source/_integrations/srp_energy.markdown [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+title: "SRP Energy"
+description: "How to integrate SRP Energy within Home Assistant."
+logo: srp_energy.png
```
Comment (klaasnicolaas) [inline source/_integrations/srp_energy.markdown]:
```suggestion
```
You can remove this line 😉

**Review by klaasnicolaas — CHANGES_REQUESTED** [review]

File: source/_integrations/srp_energy.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
+---
+title: "SRP Energy"
+description: "How to integrate SRP Energy within Home Assistant."
+ha_category:
+  - Energy
+ha_release: 0.13
```
Comment (cgarwood) [inline source/_integrations/srp_energy.markdown]: Update release to 0.119

File: source/_integrations/srp_energy.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+ha_iot_class: Cloud Polling
+---
+
+The `srp_energy` component shows information from Srp hourly energy usage report for their customers.
+
+You need a Username, Password, and AccountId which you can create at [Srp](https://www.srpnet.com).
+
+## Configuration
+
+To add Srp Energy to your installation, add the following to your `configuration.yaml` file:
```
Comment (cgarwood) [inline source/_integrations/srp_energy.markdown]: I don't think this is accurate? The integration uses a config flow, so you shouldn't need to add anything to configuration.yaml?

File: source/_integrations/srp_energy.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+  type: string
+password:
+  description: Your password for SRP.
+  required: true
+  type: string
+id:
+  description: Your account id for SRP.
+  required: true
+  type: string
+{% endconfiguration %}
```
Comment (cgarwood) [inline source/_integrations/srp_energy.markdown]: Missing information for `is_tou` / `Is Time of Use` setting from config flow

**Review by cgarwood — CHANGES_REQUESTED** [review]
Just a couple changes

File: source/_integrations/srp_energy.markdown [hunk 5]
```diff
@@ -0,0 +1,41 @@
+---
+title: "SRP Energy"
+description: "How to integrate SRP Energy within Home Assistant."
+ha_category:
+  - Energy
+ha_release: 0.119
+ha_iot_class: Cloud Polling
+---
+
+The `srp_energy` component shows information from Srp hourly energy usage report for their customers.
```
Comment (frenck) [inline source/_integrations/srp_energy.markdown]:
```suggestion
The SRP Energy integration shows information from SRP hourly energy usage report for their customers.
```

File: source/_integrations/srp_energy.markdown [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+You need a Username, Password, and AccountId which you can create at [Srp](https://www.srpnet.com).
```
Comment (frenck) [inline source/_integrations/srp_energy.markdown]:
```suggestion
You need a username, password, and account ID, which you can create at [SRP](https://www.srpnet.com).
```

File: source/_integrations/srp_energy.markdown [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+
+## Configuration
+
+Add Srp Energy to your installation from the configuraiton integration.
```
Comment (frenck) [inline source/_integrations/srp_energy.markdown]:
```suggestion
Add SRP Energy to your installation from the configuration integration.
```

File: source/_integrations/srp_energy.markdown [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+
+Navigate to `configuration`, then `integrations` and click `add`. Click `Srp Energy` to start the configuration. 
+After completing the configuration flow, the `srp_energy` integration will be available.
```
Comment (frenck) [inline source/_integrations/srp_energy.markdown]:
```suggestion
Navigate to "Configuration", then "Integrations" and click `+` button in the bottom right. Select `SRP Energy` to start the configuration. After completing the configuration procedure, the SRP Energy integration will be available.
```

File: source/_integrations/srp_energy.markdown [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+id:
+  description: Your account id for SRP.
+  required: true
+  type: string
+is_tou:
+  description: Your account is time of use planP.
+  required: false
+  type: boolean
+  default: false
+{% endconfiguration %}
```
Comment (frenck) [inline source/_integrations/srp_energy.markdown]: This is a configuration block for YAML configuration and should be removed.

File: source/_integrations/srp_energy.markdown [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+
+Details about the API are available in the [SRP Energy Developers API documentation](https://srpenergy-api-client-python.readthedocs.io/en/latest/?badge=latest).
```
Comment (frenck) [inline source/_integrations/srp_energy.markdown]: Why is this needed in the end-user documentation? What does the user get from this or need this?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/srp_energy.markdown (hunk 10 above)
Comment (briglx) [reply source/_integrations/srp_energy.markdown]: Is there a recommended way to document the fields of a config flow page?

Comment (frenck) [reply source/_integrations/srp_energy.markdown]: We currently, generally do not document those.

We have a configuration_basic block, which is currently mainly used by Lovelace UI config items that we could consider using for this purpose as well. But right now, we don't do that yet.

**Review by frenck — APPROVED** [review]
